geometric/tests/test_prepare.py

```diff
@@ -94,12 +94,40 @@ def test_parse_constraints(localizer):
     np.testing.assert_almost_equal(vals[13][24], 0.5/0.529177210903)
     np.testing.assert_almost_equal(vals[14][25], 0.0)
 
-def test_get_molecule_engine_pdb():
-    for engine, inputf in zip(['terachem', 'qchem', 'molpro', 'psi4', 'gaussian'], ['water12.tcin', 'water12.qcin', 'water12.mol', 'water12.psi4in', 'water12.gjf']):
-    # for engine, inputf in zip(['terachem', 'qchem'], ['water12.tcin', 'water12.qcin']):
-        molecule, engine = geometric.prepare.get_molecule_engine(input=os.path.join(datad, inputf), engine=engine,
-                                                                 pdb=os.path.join(datad, 'water12.pdb'), 
-                                                                 coords=os.path.join(datad, 'water12.mdcrd'))
-        assert molecule.resid[-1] == 12
-        np.testing.assert_almost_equal(molecule.xyzs[0][-1,2], 1.288)
-        assert engine.detect_dft() == False
+
+@addons.using_terachem
+def test_get_molecule_engine_pdb__terrachem():
```

Comment:
Please correct spelling to `terachem`

Reply:
thanks, fixed in 6fd00f62cdd016049c3ce1bc153d405286eafc15